dev-util/lcov/lcov-9999.ebuild

```diff
@@ -0,0 +1,37 @@
+# Copyright 1999-2018 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+if [[ ${PV} == "9999" ]] ; then
+	EGIT_REPO_URI="https://github.com/linux-test-project/lcov.git"
+	inherit git-r3
+else
+	SRC_URI="mirror://sourceforge/ltp/${P}.tar.gz"
+	KEYWORDS="~alpha ~amd64 ~arm ~arm64 ~hppa ~ia64 ~m68k ~mips ~ppc ~ppc64 ~s390 ~sh ~sparc ~x86 ~amd64-fbsd ~x86-linux ~x64-macos"
+fi
+
+inherit prefix
+
+DESCRIPTION="A graphical front-end for GCC's coverage testing tool gcov"
+HOMEPAGE="http://ltp.sourceforge.net/coverage/lcov.php"
+
+LICENSE="GPL-2"
```

Comment:
Could you verify whether this isn't `GPL-2+` actually ('version 2 or (at your option) newer')?

Reply:
I'm fairly certain the license is GPL-2. See the COPYING file in the repository: https://github.com/linux-test-project/lcov/blob/master/COPYING

Reply:
COPYING file does not define the license for the project (even though GitHub stupidly assumes so). It is only a copy of the license, while the actual license terms are listed as copyright notices on top of source files.

Reply:
It does seem that the license is `GPL-2+`. Will update.